src/Mapping/JoinTableMapping.php

```diff
@@ -84,9 +84,14 @@ public function offsetSet(mixed $offset, mixed $value): void
     /** @return mixed[] */
     public function toArray(): array
     {
-        $array = (array) $this;
+        $array                       = (array) $this;
+        $toArray                     = static function (JoinColumnMapping $column) {
+            $array = (array) $column;
 
-        $toArray                     = static fn (JoinColumnMapping $column): array => (array) $column;
+            unset($array['nullable']);
+
+            return $array;
+        };
```

Comment:
I had to do this change because association mapping overrides use it, and obtained a `nullable` property set to `false`.